components/pickerbutton/stories/template.js

```diff
@@ -1,20 +1,67 @@
 import { html } from 'lit-html';
 import { classMap } from 'lit-html/directives/class-map.js';
-// import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+
+import { Template as Icon } from "@spectrum-css/icon/stories/template.js";
 
 import "../index.css";
 
 export const Template = ({
+  id,
   rootClass = "spectrum-PickerButton",
   customClasses = [],
   size = "m",
+  isOpen = false,
+  isUIIconOnly = false,
+  isWorkflowIconOnly = false,
+  isRounded = false,
+  isInvalid = false,
+  isQuiet = false,
+  isDisabled = false,
+  label,
   // ...globals
 }) => {
+  const labelMarkup =
+    isWorkflowIconOnly === true || isUIIconOnly === true ? '' : html`<span class="${rootClass}-label is-placeholder">${label}</span>`;
+
+  const uiIconMarkup = html`
+    ${Icon({
+      iconName: "Chevron100",
```

Comment:
Don't forget to pass globals down to nested components so they are aware of platform scale and context.